corehq/apps/ota/utils.py

```diff
@@ -109,7 +109,11 @@ def is_permitted_to_restore(domain, couch_user, as_user, has_data_cleanup_privil
         if as_user is not None and not _restoring_as_yourself(couch_user, as_user):
             as_user_obj = CouchUser.get_by_username(as_user)
             if not as_user_obj:
-                raise RestorePermissionDenied(_(u'Invalid restore as user {}').format(as_user))
+                if '.commcarehq.org' not in as_user:
```

Comment:
that being said, i'm not that opposed to it given that most restore as calls are for mobile users